gremlin-python/src/main/python/gremlin_python/driver/driver_remote_connection.py

```diff
@@ -74,7 +79,19 @@ def __init__(self, url, traversal_source="g", protocol_factory=None,
         self._traversal_source = self._client._traversal_source
 
     def close(self):
-        logging.info("closing DriverRemoteConnection with url '%s'", str(self._url))
+        # close this client and any DriverRemoteConnection instances spawned from this one
+        # for a session
+        if len(self.__spawned_sessions) > 0:
+            logging.info("closing spawned sessions from DriverRemoteConnection with url '%s'", str(self._url))
+            for spawned_session in self.__spawned_sessions:
+                spawned_session.close()
+
+        if self.__session:
+            logging.info("closing DriverRemoteConnection with url '%s' with session '%s'",
+                         str(self._url), self.__session)
```

Comment:
Does this print correctly? I thought the session object in the DriverRemoteConnection class was technical a uuid object, I am wondering if self.__session should be wrapped in an `str` like so:
```suggestion
                         str(self._url), str(self.__session))
```